package.json

```diff
@@ -1,647 +1,668 @@
 {
```

Comment:
your formatter for json is using 4 spaces, vs 2.

Reply:
So weird :/
Every time I commit something, husky has been starting the linters
Anyway, I will make this changes, thank you!